### DISCUSSION TITLE
Mass Store Generator plugin

### DISCUSSION THREAD

**Discussion by pavlenex** [btcpayserver/btcpayserver#5911] [Plugins & integrations]
On a previous dev call it's been discussed that eventually between (1.14.0 and 1.15.0) we should create a plugin that allows people to mass-generate stores so they can quicker set up PoS and similar setups at scale. @Kukks has a plugin for this, but it needs re-work. 

Once it's time for this, this is potentially a plugin that @TChukwuleta and @rockstardev can look into based on Kukks scripts. However for now this is not time-sensitive and shouldn't be prioritized unless there's a immediate use-case for it.

**Comment by TChukwuleta** [comment]
Wrote a plugin for this:

https://github.com/TChukwuleta/BTCPayServerPlugins/tree/main/Plugins/BTCPayServer.Plugins.MassStoreGenerator

Would include Readme and also change the icon. But open to review
